exatomic/core/tensor.py

```diff
@@ -98,6 +101,99 @@ class Polarizability(Tensor):
                 'frame', 'label', 'type']
     _categories = {'frame': np.int64, 'label': str}
 
+class NMRshielding(Tensor):
```

Comment:
It might be outside the scope of this PR but I would have thought that `NMRShielding` (can we use that capitalization) would inherit from a class called `Tensor2` or something similar to indicate the base class as dealing with tensors of rank (order) 2 (the class `Tensor2` would itself inherit from `Tensor`). Even if we don't have the details ironed out now, I think that abstraction provides future flexibility.

Reply:
Yeah, I have been thinking about that. It should be easy enough to at least change the names.
I definitely think we could do something with the tensors. I have been starting to do a lot more with parsing tensors and it may be good to have something like that.